src/sentry/api/endpoints/organization_events.py

```diff
@@ -678,7 +686,7 @@ def data_fn_factory(scoped_dataset):
             dashboard_widget_id = request.GET.get("dashboardWidgetId", None)
             discover_saved_query_id = request.GET.get("discoverSavedQueryId", None)
 
-            def fn(offset, limit) -> dict[str, Any]:
+            def fn(offset, limit) -> Mapping[str, Any]:
```

Comment:
generally functions should return *concrete* types (same above): https://typing.readthedocs.io/en/latest/reference/best_practices.html#arguments-and-return-types